### PR TITLE
Docker dashboard updates & fixes

### PR DESCRIPTION
**Pull request by mshahzeb**
Docker dashboard updates and fixes based on feedback from: https://github.com/grafana/jsonnet-libs/issues/754

- [ ] (Reverted this is done at the integrations level already) Add filter to prometheus variable for cloud-usage datasources
- [x] Update variable display names
- [x] Rename datasource to prometheus_datasource
- [x] Group by instance as well on disk usage
- [x] Fix total images / containers showing up as double
- [x] Add integration status panels
- [x] Convert to gridpos
- [x] Remove deprecated singlestat panel
- [x] Disable shared tooltip

### CODE REVIEW

**Comment by mshahzeb** [comment]
I am not sure why it is giving a conflict since I am updating the file completely. @rgeyer How do you suggest I resolve the conflict? Just remove the part from the master branch?

**Comment by mshahzeb** [comment]
Updated dashboard screenshot

image

**Comment by rgeyer** [comment]
Looks like you were working off of an older version of master. In #755 some changes were made (from `#` to `//` for comments) on a few files, including the `docker.libsonnet`.

You should be able to resolve by doing a;

```
git pull master
git rebase master
<resolve conflicts as necessary>
git push --force
````

**Comment by mshahzeb** [comment]
Thanks @rgeyer ! I have resolved the conflicts and updated.

File: docker-mixin/dashboards/docker.libsonnet [hunk 1]
```diff
@@ -35,18 +35,19 @@ local ds_template = {
   },
   hide: 0,
   label: 'Data Source',
-  name: 'datasource',
+  name: 'prometheus_datasource',
   options: [],
   query: 'prometheus',
   refresh: 1,
-  regex: '',
+  regex: '(?!grafanacloud-usage|grafanacloud-ml-metrics).+',
```
Comment (RutgerKe) [inline docker-mixin/dashboards/docker.libsonnet:42]: This seems grafana cloud specific, and should probably not be done in this lib? I think this is already taken care of [here for cloud](https://github.com/grafana/cloud-onboarding/blob/7588a0520269d704d2cdb251c2cf48f1728d9c95/integrations/util.libsonnet#L15).

Comment (mshahzeb) [reply docker-mixin/dashboards/docker.libsonnet:42]: Thanks @RutgerKe ! You are right - I will remove this.

**Review by rgeyer — APPROVED** [review]
One pedantic nit below RE: datasource variable naming. Otherwise, LGTM!

File: docker-mixin/dashboards/docker.libsonnet [hunk 2]
```diff
@@ -35,7 +35,7 @@ local ds_template = {
   },
   hide: 0,
   label: 'Data Source',
-  name: 'datasource',
+  name: 'prometheus_datasource',
```
[the remark this answers is not in the document]
Comment (mshahzeb) [reply docker-mixin/dashboards/docker.libsonnet:38]: Thanks @rgeyer ! I have made the latest commit to add lint exclusion file.

**Pull request merged**